Gaijinizer patch/Map308.txt

```diff
@@ -127,10 +125,9 @@ Let's withdraw quickly.#GT
  お前が、この世界のユウマにでも見つかったら
  まーた話がややこしくなっちまうからな……
 > CONTEXT: Map308/events/5/pages/0/65/Dialogue/other1 
-Because you are "existence of interventions
-that embody"
-even if you are found in this world of Yuuma,
-the story will become more complicated......#GT
+Since you're just an "Intervention body,"
+Getting found even just by Yuuma could really
+mess this world up after all...#Amaboye
 > END STRING
```

Comment:
Revising the text and adding a bit of localization to make this clearer:
```
Since you're an "intervening" body... an outsider...
getting found by the Yuuma of this world could
cause unwanted complications...
```